packages/blade/src/components/DatePicker/_decisions/decisions.md

```diff
@@ -0,0 +1,352 @@
+# DatePicker
+
+The Date Picker component lets users select a date from a calendar. It is used to input a date or range of dates.
+
+<img src="./datepicker-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link]() to all variants of the DatePicker component
+
+## Anatomy
+
+<img src="./datepicker-anatomy.png" width="100%" alt="DatePicker Anatomy" />
+
+## Components
```

Comment:
So is this assumption correct:
- Calendar is an independent component that doesn't need a trigger
- DatePicker is a component that comes with a trigger upon clicking which a calendar is popped up

Reply:
Yes. 

Tho we will not expose the Calendar component.